### PR TITLE
Install cypress

### PR DESCRIPTION
**Pull request by Antonieta13**
Hi guys

Can you review this pull request, please?
 
Waiting for your responses

Thank you

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,5 +21,8 @@
   "bugs": {
     "url": "https://github.com/Antonieta13/cypress-training/issues"
   },
-  "homepage": "https://github.com/Antonieta13/cypress-training#readme"
-}
+  "homepage": "https://github.com/Antonieta13/cypress-training#readme",
+  "devDependencies": {
+    "cypress": "^6.8.0"
+  }
+}
```
Comment (holgiosalos) [inline package.json]: EOF

Comment (Antonieta13) [reply package.json:28]: Hola Holmes no termino de entender el EOF, es en la linea 28? yo en el vscode 
coloco enter y me salta a la linea 29 pero al guardar se regresa a la 28

Comment (Antonieta13) [reply package.json]: Fixing EOF

**Review by holgiosalos — APPROVED** [review]

**Comment by holgiosalos** [comment]
Please remember:

1. Squash and merge
2. Checkout to master in your local
3. Git pull (to update your local to the latest changes) 
4. Create the new branch for the next step after pull is done 

image

**Pull request merged**